config/settings.yml

```diff
@@ -806,7 +806,6 @@ oidc:
     veteran_verification: ~
     claims: ~
   charon:
```

Comment:
The Charon call is specifically for SSOi OAuth validation events, and once TV is removed here these settings are no longer applicable.

We will want corresponding removals in the manifests repo
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vets-api/

Reply:
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2293
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2294